src/stream/src/task/mod.rs

```diff
@@ -40,16 +41,23 @@ pub type UpDownActorIds = (ActorId, ActorId);
 pub type UpDownFragmentIds = (FragmentId, FragmentId);
 
 #[derive(Hash, Eq, PartialEq, Copy, Clone, Debug)]
-struct PartialGraphId(u64);
+pub(crate) struct PartialGraphId(u32);
```

Comment:
I recall we used to have database id part of the partial graph id https://github.com/risingwavelabs/risingwave/pull/19173#discussion_r1830932103.

I suppose now that we always have a separate graph per db, this is no longer necessary.

Reply:
Yes. partial graph id should be unique only under a same database, and does not require to be unique globally.